components/record/MediaThumbnailGrid.vue

```diff
@@ -1,19 +1,29 @@
 <template>
   <section
     data-qa="media thumbnail grid"
+    class="d-flex flex-wrap mb-3"
   >
     <!-- TODO: populate alt, but with what? -->
     <b-img
       v-for="(thumbnail, index) of thumbnails"
       :key="index"
       :src="thumbnail.src"
       :class="{ 'selected' : isSelected(thumbnail) }"
+      :style="{ 'display' : index > 10 && !showAll ? 'none' : 'inline' }"
```

Comment:
Probably be cleaner to add a class to hide the image `.d-none` https://getbootstrap.com/docs/4.0/utilities/display/#hiding-elements